lib/plugins/aws/package/compile/events/websockets/lib/api.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
           Name: this.provider.naming.getWebsocketsApiName(),
           RouteSelectionExpression,
           Description:
-            this.serverless.service.provider.websocketsDescription || 'Serverless Websockets',
```

Comment:
We should still support the old syntax, but checking the new one first when resolving the property